pkg/exec/helpers.go

```diff
@@ -132,3 +133,23 @@ func RunWithStdinWriter(cmd Cmd, writerFunc func(io.Writer) error) error {
 		},
 	})
 }
+
+// ToPath is a helper to run a cmd and write the output to path
+func ToPath(cmd Cmd, path string) error {
+	if err := os.MkdirAll(filepath.Dir(path), os.ModePerm); err != nil {
+		return err
+	}
+	f, err := os.Create(path)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+	cmd.SetStdout(f)
+	cmd.SetStderr(f)
+	return cmd.Run()
+}
+
+// ToPathFn returns the ToPath function, useful for running concurrently
```

Comment:
I don't think this should be in exec. it's just a trivial lambda that can be done inline